src/plugins/data_source_management/public/components/cluster_selector/cluster_selector.tsx

```diff
@@ -60,7 +61,9 @@ export class ClusterSelector extends React.Component<ClusterSelectorProps, Clust
             label: dataSource.title,
           }));
 
-          clusterOptions.push(LocalCluster);
+          if (!this.props.hideLocalCluster) {
+            clusterOptions.unshift(LocalCluster);
```

Comment:
if not hiding/showing local cluster, the default state at line 45 already has local cluster, are we adding it again?

Reply:
Thanks for the comment.

We are not adding it again.
The clusterOptions array is newly created at line 59~